### PR TITLE
Fixed home icon not sending to correct location, added to interfaces

### PR DESCRIPTION
**Pull request by CBColleg3**

### CODE REVIEW

File: client/src/common/types/Course.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@
  */
 interface Course {
     name: string;
+    courseID: string;
+    description: string;
+    credithours: number;
```
Comment (cthacker-udel) [inline client/src/common/types/Course.ts:8]: Just name this to credits?

File: client/src/common/types/Course.ts [hunk 2]
```diff
@@ -3,6 +3,11 @@
...
+    prereq:  string[];
```
Comment (cthacker-udel) [inline client/src/common/types/Course.ts:9]: The data represents it as a string, should we make it a string array?

Comment (cthacker-udel) [reply client/src/common/types/Course.ts:9]: What I mean by that, is that should we take it in as a string, and then when we need to utilize it, we split it into an array.

File: client/src/common/types/Course.ts [hunk 3]
```diff
@@ -3,6 +3,11 @@
...
+    satisfied_reqs: string[];
```
Comment (cthacker-udel) [inline client/src/common/types/Course.ts:10]: What is this for?

File: client/src/common/types/Course.ts [hunk 4]
```diff
@@ -3,6 +3,11 @@
...
     title?: string;
```
Comment (cthacker-udel) [inline client/src/common/types/Course.ts:11]: Isn't that the name?

File: client/src/common/types/Semester.ts [hunk 5]
```diff
@@ -4,6 +4,10 @@ import type Course from "./Course";
  * Semester type, contains a list of courses
  */
 interface Semester {
+    term: string;
```
Comment (cthacker-udel) [inline client/src/common/types/Semester.ts:7]: Make this an number, I'd rather the seasons be represented as ints, like 0 for spring, 1 for summer, 2 for fall, 3 for winter

File: client/src/locale/en/home.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
     "card_header": "This is a planner to help you schedule your courses for whichever plan you decide on doing.",
     "card_title": "Options (click each section to see more)",
-    "created_by": "Created by {name1}.",
+    "created_by": "Created by {name1} and Christopher Bennett.",
```
Comment (cthacker-udel) [inline client/src/locale/en/home.json:4]: 👍

File: client/src/modules/common/components/Layout/Layout.tsx [hunk 7]
```diff
@@ -255,7 +255,7 @@ export const Layout = (): JSX.Element => {
     ];
 
     return (
-        <>
```
Comment (cthacker-udel) [inline client/src/modules/common/components/Layout/Layout.tsx:258]: This should be original, `<>` and `</>` respectively

**Review by cthacker-udel — CHANGES_REQUESTED** [review]

File: client/src/common/types/Course.ts (hunk 1 above)
Comment (cthacker-udel) [reply client/src/common/types/Course.ts:8]: ^^ `credithours: number;`

File: client/src/modules/common/components/Layout/Layout.tsx (hunk 7 above)
Comment (CBColleg3) [reply client/src/modules/common/components/Layout/Layout.tsx:258]: Where? shouldnt it be a div?

File: client/src/common/types/Course.ts (hunk 3 above)
Comment (CBColleg3) [reply client/src/common/types/Course.ts:10]: Those are the university requirements you will be satisfying by taking that course, i.e. Capstone series, or breadth requirements

File: client/src/common/types/Course.ts (hunk 2 above)
Comment (CBColleg3) [reply client/src/common/types/Course.ts:9]: Sure we can do that

File: client/src/common/types/Course.ts (hunk 4 above)
Comment (CBColleg3) [reply client/src/common/types/Course.ts:11]: I didn't know what title was so I guess so.

File: client/src/common/types/Course.ts (hunk 1 above)
Comment (cthacker-udel) [reply client/src/common/types/Course.ts:8]: Address this

File: client/src/modules/common/components/Layout/Layout.tsx (hunk 7 above)
Comment (cthacker-udel) [reply client/src/modules/common/components/Layout/Layout.tsx:258]: I'll just approve for now and change it later, won't matter too much anyway.

**Review by cthacker-udel — APPROVED** [review]
Good work!

**Pull request merged**